### ISSUE TITLE
Place editor groups side-by-side with The Panel

### ISSUE THREAD

**Issue by PEZ** [microsoft/vscode#68831]
I want to be able to split the window more freely. Specifically when using a small screen it gets hard to follow the output of some extension or task, or something in the terminal while editing a file and having  another file split open for reference. Here's what that can look like on a small screen:

![image](https://user-images.githubusercontent.com/30010/52904508-bbf7b100-322d-11e9-9d91-1f127402e0a2.png)

Moving the panel to the right does not really help:

![image](https://user-images.githubusercontent.com/30010/52911600-29472880-32a6-11e9-891a-dacc636caaa6.png)

If I could move an editor group down side by side with the panel (or below or under it when it is placed to the right) this would save me a lot of moving around between tabs and groups.

This gets extra important with some extensions that contribute WebViews with much of their UI. They are sometimes more panel-like than editor-like.

With this feature added screen real estate could be used like so:

![bitmap](https://user-images.githubusercontent.com/30010/52904570-08db8780-322e-11e9-96da-b3e0cffacce0.png)

Or with the panel to the right:

![bitmap](https://user-images.githubusercontent.com/30010/52904578-1bee5780-322e-11e9-8ecb-11f6c21c6d5f.png)

I think that it could be made general so that also the explorer pane could participate in this sharing.

Let me know if my suggestion is not clear enough, and I'll try to improve on it.

**Comment by PEZ** [comment]
Is the description of the feature request clear enough? Basically I would like to be able to place any pane anywhere.

**Comment by Cephastorm** [comment]
I would very much like for this to be a thing! I seldom need the full width (or height) of the terminal/browser and placing it side by side with another window would considerably help my workflow.
